collections/README.md

```diff
@@ -509,3 +509,19 @@ console.assert(
   ],
 );
 ```
+
+### includesValue
+
+Returns whether the given value is part of the collection.
```

Comment:
We should change that to something meaningful. Imagine knowing nothing about this function and reading this for the first time.

Reply:
I'm not the best at documentation in all honesty, how does `If the given value is part of the given object it returns true, otherwise it returns false.` sound?

Reply:
You can do `Readonly<Record<>>`

Reply:
Returns whether the given value is in the given record's values ?

Reply:
I've pushed the above but will leave this unresolved for further discussion.

Reply:
> Returns whether the given value is in the given record's values ?

Hmm I guess we could do that but imo what is currently rn is more detailed, i guess i could change `object` to `record` but it doesn't make much difference.